README.md

```diff
@@ -1,70 +1,61 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
+[![Netlify Status](https://api.netlify.com/api/v1/badges/e0f14414-e46b-4264-85f4-02a55abbb7e4/deploy-status)](https://app.netlify.com/sites/remarkable-cupcake-36a101/deploys)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Crypto Bazaar
 
-## Available Scripts
+This is a React app that uses the [Coin-Cap API](https://docs.coincap.io/). Displays over a 1000 crypto currencies, their current values and graph of their changes over the recent days.
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- [React](https://reactjs.org/)
+- [Redux](https://redux.js.org/)
+- [React-Router](https://reactrouter.com/)
+- Chart.js
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo
 
```

Comment:
The video presentation about your project is missing! Kindly endeavor to record a maximum of 5 minutes video presenting the features of the project in the ReadMe or within your Pull Request project details/comment. The recorded video should:
- [ ]  entail the **features** of the project.
- [ ] have a maximum length of **5 minutes**.
- [ ] be **clear and objective description** about the project in professional English. The topic and purpose of the project are made clear.
- [ ] contains a demo of the project, highlighting all the project features.
- [ ] contain a highlight about a piece of the code the student finds interesting or is proud of making.
-